Bluetooth: Mesh: Fixing errors in Mesh Sample Documation (#3957)
nrfconnect/sdk-nrf: samples/bluetooth/mesh/light_ctrl/README.rst

See :ref:`ug_bt_mesh_model_config_app` for details on how to configure the mesh models with the nRF Mesh mobile app.

Configure the Generic OnOff Server model on each element on the :guilabel:`Mesh Light Fixture` node:

* Bind the model to :guilabel:`Application Key 1`.

  Once the model is bound to the application key, you can control the first LED on the device.

> * In the model view, tap :guilabel:`ON` on the second Generic On Off Server to light up the first LED on the development kit.

Reviewer: ```suggestion
* In the model view of the Generic OnOff Server in the second element, tap :guilabel:`ON`.
```

The resulting sequence is described below.

Reviewer 2: I still kind of find it a bit hard to read.

I haven't tested this myself, so I am not familiar with the UI you see when you are actually doing this. Do you select the element first? Or do you go to the model view, see the model (OnOff Server) and select what element? Is it possible to send me a screen shot, so I know what we are describing?

Author: You are correct. This can be made more clear. I'll come up with a suggestion.

Reviewer 2: Thanks for the screen shots and the nice discussion, I like the solution.

I wonder if Trond meant that "You should now see the following actions:" should be replaced with "The resulting sequence is described below."? It's the only thing left then.

You should now see the following actions:

1. The LED fades from 0% to 100% over 500ms :guilabel:`Standby -> On`.
#. The LED stays at 100% for three seconds :guilabel:`On`.
#. The LED fades from 100% to CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG over five seconds :guilabel:`On -> Prolong`.
#. The LED stays at CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG for three seconds :guilabel:`Prolong`.
#. The LED fades from CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG to 0% over five seconds :guilabel:`Prolong -> Standby`.

> CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG has 10000 as default value.

Reviewer: Use ``:option:`CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG` `` when referring to configuration options.
Also, perhaps this sentence should be

> The default value of :option:\`CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG\` is 15% (10000)
